droidconKE/chai
# prevent crashing because of unremoved properties

---
name: Dependency Updates
about: Updating changes to the project from dependabot
label: 'triage me'
Fixes #3 (Recurring prs)


---
## How can this PR be tested?
RU\un the project and it should work better

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:


Is this your first Pull Request?

Fixes #<issue_number_goes_here> 🦕

# Screenshot
> For PRs that change the UI, attach a before/after screenshot.
> **[REMOVE THIS SECTION IF IT DOES NOT APPLY TO THIS PR]**

![]()
